conflicts with ebuild-applied patch: 1.46.0-don-t-create-prefix-at-time-of-check.patch

#index 2d040d60e5..88f2434871 100644
#@@ -68,7 +68,7 @@ fn install_sh(
#     let libdir_default = PathBuf::from("lib");
#     let mandir_default = datadir_default.join("man");
#     let prefix = builder.config.prefix.as_ref().map_or(prefix_default, |p| {
#-        fs::canonicalize(p).unwrap_or_else(|_| panic!("could not canonicalize {}", p.display()))
#+        fs::canonicalize(p).unwrap_or_else(|_| panic!("could not canonicalize path in config.toml [install] prefix=\"{}\". Please make sure you `make -p -- path` so that it already exists before `./x.py install` is called.", p.display()))
#     });
#     let sysconfdir = builder.config.sysconfdir.as_ref().unwrap_or(&sysconfdir_default);
#     let datadir = builder.config.datadir.as_ref().unwrap_or(&datadir_default);

Index: /var/tmp/portage/dev-lang/rust-9999/work/rust-git-src/src/bootstrap/install.rs
===================================================================
diff --git a/src/bootstrap/install.rs b/src/bootstrap/install.rs
--- a/src/bootstrap/install.rs
+++ b/src/bootstrap/install.rs
@@ -77,7 +77,7 @@ fn install_sh(
         fs::create_dir_all(p)
             .unwrap_or_else(|err| panic!("could not create {}: {}", p.display(), err));
         fs::canonicalize(p)
-            .unwrap_or_else(|err| panic!("could not canonicalize {}: {}", p.display(), err))
+            .unwrap_or_else(|err| panic!("could not canonicalize path in config.toml [install] prefix=\"{}\". Err='{}'. Please make sure you `make -p -- path` so that it already exists before `./x.py install` is called.", p.display(), err))
     });
     let sysconfdir = builder.config.sysconfdir.as_ref().unwrap_or(&sysconfdir_default);
     let datadir = builder.config.datadir.as_ref().unwrap_or(&datadir_default);
